File: 251012/좌우로 반복하며 출력/print-in-left-right-reverse.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int a;
    cin >> a;

    int array[a][a];
    

    for(int i = 0; i < a; i++) {
        if(i % 2 != 0) {
            int count = a;
            for(int j = 0; j < a; j++) {
                cout << count;
                count--;
            }
            } else {
            int count = 1;
                for(int j = 0; j < a; j++) {
                    cout << count;
                    count++;
                }
         }
         cout << "\n";
        }
        
        return 0;

    }

File: 251111/좌우로 반복하며 출력/print-in-left-right-reverse.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        // Please write your code here.
        int a = int.Parse(Console.ReadLine());
        for(int i = 0; i < a; i++) {
            for(int j = 0; j < a; j++) {
                if(i % 2 == 0) {
                    Console.Write(j+1);
                } else {
                    Console.Write(a-j);
                }
            }
            Console.WriteLine("");
        }
    }
}
